CODE_OF_CONDUCT.rst

```diff
@@ -0,0 +1,86 @@
+# Code of Conduct
+
+## 1. Purpose
+
+A primary goal of Kolibri is to be inclusive to the largest number of contributors, with the most varied and diverse backgrounds possible. As such, we are committed to providing a friendly, safe and welcoming environment for all, regardless of gender, sexual orientation, ability, ethnicity, socioeconomic status, and religion (or lack thereof).
```

Comment:
> A primary goal of Kolibri is to be inclusive to the largest number of contributors, with the most varied and diverse backgrounds possible.

A couple notes:
* If we're using the same code of conduct across the organization, I would recommend changing "Kolibri" to "Learning Equality"
* The phrasing is a little confusing - it uses the superlative forms ("largest", "most"), but then hedges with a somewhat ambiguously associated "possible".

Perhaps something more simple, e.g.:

> A primary goal of Learning Equality is to be as inclusive as possible to contributors from varied and diverse backgrounds.

or even distill it down to just:

> A primary goal of Learning Equality is to be as inclusive as possible.